userspace/otpilot/src/spi_device.rs

```diff
@@ -0,0 +1,170 @@
+// Copyright 2020 lowRISC contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use core::cell::Cell;
+
+use libtock::result::TockResult;
+use libtock::shared_memory::SharedMemory;
+use libtock::syscalls;
+use libtock::syscalls::raw::yieldk;
+
+pub const MAX_READ_BUFFER_SIZE : usize = 512;
+
+#[allow(dead_code)]
+pub const MAX_WRITE_BUFFER_SIZE : usize = 2048;
```

Comment:
Nitpick: Remove the unnecessary space before the `:` on MAX_READ_BUFFER_SIZE and MAX_WRITE_BUFFER_SIZE.

Reply:
Done.